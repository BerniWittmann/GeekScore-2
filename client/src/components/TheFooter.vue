<template>
  <v-footer class="the-footer" padless>
    <v-layout primary white--text>
      <v-container>
        <span class="mr-6" data-test-copy>&copy; {{ copy }}</span>
        <span>
          Build by
          <a class="link" href="https://twitter.com/david_go__" target="_blank">@David_Go__</a>
        </span>
      </v-container>
    </v-layout>
  </v-footer>
</template>

<script>
export default {
  computed: {
    year() {
      const date = new Date();
      return date.getFullYear();
    },
    copy() {
      return ` ${this.year} â€” Geek Score`
    }
  }
};
</script>

<style scoped lang="scss">
.the-footer {
  .layout {
    justify-content: space-between;
    flex-wrap: wrap;
    padding: 8px 0;
  }
  .container {
    display: flex;
    justify-content: space-between;
    flex-wrap: wrap;
    max-width: $app-max-width;
    font-size: 12px;
    @media #{$tablet} {
      font-size: 14px;
    }
  }
  .link {
    color: $secondary;
    margin-left: 5px;
    text-decoration: none;
  }
}
</style>