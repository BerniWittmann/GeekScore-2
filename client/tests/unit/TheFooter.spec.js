import TheFooter from "@/components/TheFooter"
import { mount, shallowMount } from '@vue/test-utils'

describe('TheFooter', () => {
    it('Shallow test', async () => {
        const wrapper = mount(TheFooter)
        expect(wrapper.html()).toMatchSnapshot()
    })
})