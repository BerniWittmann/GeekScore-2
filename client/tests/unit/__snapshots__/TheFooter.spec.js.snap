// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooter Shallow test 1`] = `
<v-footer padless="" class="the-footer">
  <v-layout justify-space-between="" py-2="" wrap="" primary="" text-center="" white--text="" xs12="">
    <v-container class="d-flex flex-wrap justify-space-between footer-text"><span class="mr-6">© 2020 — Geek Score</span> <span>geekscore.info@gmail.com</span></v-container>
  </v-layout>
</v-footer>
`;
